templates/index.tpl

```diff
@@ -20,7 +20,7 @@
             {if $config->convert}
                 <div class="mp3 small-font">
                     <div class="mp3-inner">
-                        <input type="checkbox" id="audio" class="audio" name="audio">
+                        <input type="checkbox" id="audio" class="audio" name="audio"{($config->defaultAudio) ? ' checked' : ''}>
```

Comment:
I would find this a bit easier to read:
```suggestion
                        <input type="checkbox" id="audio" class="audio" name="audio" {($config->defaultAudio) ? 'checked' : ''}>
```
What do you think?

Reply:
If the html can handle it - definitely.